### ISSUE TITLE
Compile features using exact same dataset settings

### ISSUE THREAD

**Issue by siemdejong** [siemdejong/sclicom#32]
**Describe the bug**
The features may not have to be compiled using exactly the same settings for the datamodule as on which the feature extractor is trained.
This is a problem, as it might perform bad on images without a mask or tiles with a different size or mpp.

**To Reproduce**
1. Train a feature extractor
2. Compile features with a new `PMCHHGImageDataset`

**Desktop (please complete the following information):**
 - DPAT version 4.3.2

**Additional context**
A solution might be to save the pytorch lightning hyperparameters with `save_hyperparameters` or ask for the specific config file the feature extractor trained with, and use those variables for instantiation of `PMCHHGImageDataset`
